libr/bin/bfile.c

```diff
@@ -856,24 +856,17 @@ R_API bool r_bin_file_close(RBin *bin, int bd) {
 	return false;
 }
 
-R_API RList *r_bin_file_hash(RBin *bin, ut64 limit) {
-	r_return_val_if_fail (bin, NULL);
-
-	char hash[128];
-	RHash *ctx;
+R_API RList *r_bin_file_compute_hashes(RBin *bin, ut64 limit) {
+	r_return_val_if_fail (bin && bin->cur && bin->cur->o, NULL);
```

Comment:
Actually this may happen sometimes, it should be checked.
See https://travis-ci.com/radareorg/radare2/jobs/289601576#L739 . I'm not sure why it would happen though.

Same for `r_bin_file_set_hashes`.

Reply:
I found out that those tests use the `-n` flag, so `r_bin` analysis is not performed and `r->bin` will be null.

Reply:
but it should output to `stderr` instead of `stdout` since it is a `assert` (?)

Reply:
I think these checks were right with just an if then.

In general assert prints to stderr but in Travis we use an env variable to redirect that to stdout so we can catch these kind of problems